nikosnick13.HabitLogger/Program.cs

```diff
@@ -0,0 +1,293 @@
+﻿using Microsoft.Data.Sqlite;
+using System.Data;
+using System.Globalization;
+using static System.Console;
+
+namespace nikosnick13.HabitLogger
+{
+    internal class Program
+    {
+
+
+        static string connectionString = @"Data Source=habit-Traker.db";
+
+        static void Main(string[] args)
+        {
+
+
+            using (var conn = new SqliteConnection(connectionString))
+            {
+                conn.Open();
+
+                var tableCmd = conn.CreateCommand();
+
+                tableCmd.CommandText =
+                    @"CREATE TABLE IF NOT EXISTS  dring_water(
+                        Id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        Date TEXT, 
+                        Quantity  INTEGER
+                    )";
+
+                tableCmd.ExecuteNonQuery();
+
+                conn.Close();
+
+
+
+            }
+
+            UserInput();
+
+            ReadKey();
+
+        }
+
+        //USER INPUT MENY
+        public static void UserInput()
+        {
+            Clear();
+            ForegroundColor = ConsoleColor.Green;
+
+            bool isAppRunning = false;
+
+            while (!isAppRunning)
+            {
+               
+                WriteLine("\n\nMAIN Menu");
+                WriteLine("\nWhat would you like to do?");
+                WriteLine("\n\tType 0 to Close Application.");
+                WriteLine("\tType 1 to Inser a Record ");
```

Comment:
Typo:
`WriteLine("\tType 1 to Inser a Record ");`

`WriteLine("\tType 1 to Insert a Record ");`